styleguide/sample-pages/advice-collection/_advice_collection_adviser.html.haml

```diff
@@ -27,6 +27,26 @@
                 %p The important callout should be used for any important snippet of text that has serious and/or legal implications if the client does not follow the advice.
             %p
               Every page should have some lorem ipsum. So here it is.
+            %ul
```

Comment:
Maybe we should create a "kitchen sink" sample page with this and a few more prose specific examples?

Reply:
I think the public advice collection one covers most things, I've added lists to that one too.  We can add more tho!